docs/core/extensions/custom-logging-provider.md

```diff
@@ -38,26 +38,27 @@ The `ILoggerProvider` object is responsible for creating logger instances. Maybe
 
 :::code language="csharp" source="snippets/configuration/console-custom-logging/ColorConsoleLoggerProvider.cs":::
 
-In the preceding code, <xref:Microsoft.Build.Logging.LoggerDescription.CreateLogger%2A> creates a single instance of the `ColorConsoleLogger` per category name and stores it in the [`ConcurrentDictionary<TKey,TValue>`](/dotnet/api/system.collections.concurrent.concurrentdictionary-2).
+In the preceding code, <xref:Microsoft.Build.Logging.LoggerDescription.CreateLogger%2A> creates a single instance of the `ColorConsoleLogger` per category name and stores it in the [`ConcurrentDictionary<TKey,TValue>`](/dotnet/api/system.collections.concurrent.concurrentdictionary-2). Additionally, the <xref:Microsoft.Extensions.Options.IOptionsMonitor%601> interface is required to update changes to the underlying `ColorConsoleLoggerConfiguration` object.
 
 ## Usage and registration of the custom logger
 
 To add the custom logging provider and corresponding logger, add an <xref:Microsoft.Extensions.Logging.ILoggerProvider> with <xref:Microsoft.Extensions.Logging.ILoggingBuilder> from the <xref:Microsoft.Extensions.Hosting.HostingHostBuilderExtensions.ConfigureLogging(Microsoft.Extensions.Hosting.IHostBuilder,System.Action{Microsoft.Extensions.Logging.ILoggingBuilder})?displayProperty=nameWithType>:
 
-:::code language="csharp" source="snippets/configuration/console-custom-logging/Program.cs" range="23-39":::
+:::code language="csharp" source="snippets/configuration/console-custom-logging/Program.cs" range="23-33":::
 
 The `ILoggingBuilder` creates one or more `ILogger` instances. The `ILogger` instances are used by the framework to log the information.
 
-For the preceding code, provide at least one extension method for the `ILoggerFactory`:
+By convention, extension methods on `ILoggingBuilder` are used to register the custom provider:
 
 :::code language="csharp" source="snippets/configuration/console-custom-logging/Extensions/ColorConsoleLoggerExtensions.cs":::
 
-Running this simple application will render similar to the following console window:
+Running this simple application will render similar color output to the console window:
 
 :::image type="content" source="media/color-console-logger.png" alt-text="Color console logger sample output":::
 
 ## See also
 
 - [Logging in .NET](logging.md).
 - [Logging providers in .NET](logging-providers.md).
+- [Dependency injection in .NET](dependency-injection.md).
```

Comment:
I've never seen periods used on see also bullets before.

Reply:
I thought @tdykstra shared with me recently that there was guidance around doing this - can't recall. I removed them.

Reply:
That's when the intro phrase and bullet text make a sentence.  See-also links typically aren't sentences and so don't get periods.